.github/workflows/release.yml

```diff
@@ -20,10 +20,50 @@ on:
   workflow_dispatch:
 
 jobs:
-  dummy:
+  wait-for-images:
     runs-on: ubuntu-latest
-    name: Dummy job
+    name: Wait for container images
 
     steps:
-      - name: Dummy
-        run: echo "Dummy step"
+      - name: Wait for build-base-images workflow to succeed
+        uses: fountainhead/action-wait-for-check@v1.1.0
+        with:
+          checkName: Building image
+          token: ${{ secrets.GITHUB_TOKEN }}
+          timeoutSeconds: 5400
```

Comment:
9h timeout? seems quite long. 3h should be enough or?

Reply:
5400 seconds / 60 = 90minutes

Reply:
oh man, you are right... forget it :)